src/app/containers/Brand/index.stories.jsx

```diff
@@ -1,24 +1,31 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
 import * as svgs from '@bbc/psammead-assets/svgs';
+import { inputProvider } from '@bbc/psammead-storybook-helpers';
+import { withKnobs } from '@storybook/addon-knobs';
+import services from '../../lib/config/services';
 import { ServiceContextProvider } from '../../contexts/ServiceContext';
 import Brand from '.';
-import services from '../../lib/config/services';
-
-const stories = storiesOf('Brand', module);
 
-Object.keys(services)
-  .filter(service => service !== 'default')
-  .forEach(service => {
-    stories.add(service, () => (
-      <ServiceContextProvider service={service}>
-        <Brand
-          product="BBC News"
-          svg={svgs[service]}
-          svgHeight={24}
-          maxWidth={280}
-          minWidth={180}
-        />
-      </ServiceContextProvider>
-    ));
-  });
+storiesOf('Brand', module)
+  .addDecorator(withKnobs)
+  .add(
+    'default',
+    inputProvider(
+      null,
+      ({ service }) => {
+        return (
+          <ServiceContextProvider service={service}>
+            <Brand
```

Comment:
Not sure this is right, the brand container only appears to take 2 props as far as I can tell https://github.com/bbc/simorgh/blob/latest/src/app/containers/Brand/index.jsx#L30-L33 so in storybook the new brand appears further right than the others

Reply:
nice spot, i was just mindlessly updating the previous implementation of it. 
![mindless](https://media.giphy.com/media/BcJbdfFSsJPwK4O66J/giphy.gif)